src/Consolonia.Gallery/View/ControlsListView.axaml.cs

```diff
@@ -9,6 +9,7 @@
 using Avalonia.Controls.ApplicationLifetimes;
```

Comment:
Strange coloring in the settings. Text is supposed to be bright (on dark background). Need investigation
<img width="1471" height="1026" alt="image" src="https://github.com/user-attachments/assets/643d531e-267f-41ac-9814-497ff804fd2b" />

Reply:
This happens in Modern Dark theme. Font is supposed to be bright, not dark, because background is dak.

Reply:
Will address this separately